.github/dependabot.yml

```diff
@@ -14,6 +14,21 @@ updates:
       - "raft review"
     reviewers:
       - "raft-tech/tdrs-developers"
+    ignore:
+      - dependency-name: "redux*"
+        update-types: ["version-update:semver-patch"]
+      - dependency-name: "cypress*"
+        update-types: ["version-update:semver-patch"]
+      - dependency-name: "prettier*"
+        update-types: ["version-update:semver-patch"]
+      - dependency-name: "eslint*"
+        update-types: ["version-update:semver-patch"]
+      - dependency-name: "react-dom*"
+        update-types: ["version-update:semver-patch"]
```

Comment:
Nit, but `react-dom` shouldn't be ignored here, in my opinion.

Reply:
Thanks, will change